generators/wp/templates/chisel-starter-theme/functions.php

```diff
@@ -166,6 +172,26 @@ public function twig_class_name($name = '') {
 		}
 		return implode(' ', $classes);
 	}
+
+	/**
+	 * Verifies existence of the vendor.js file
+	 *
+	 * @return bool
+	 */
+	public function twig_has_vendor () {
+		if ( file_exists( get_template_directory() . '/dist/scripts/vendor.js' ) ) {
+			return true;
+		}
+
+		if ( ! defined( 'CHISEL_DEV_ENV' ) && file_exists( get_template_directory() . '/' . $this->manifestPath ) ) {
+			$manifest = json_decode( file_get_contents( get_template_directory() . '/' . $this->manifestPath ),
+				true );
```

Comment:
Is there a reason to parse manifest here, why not use the same parsed manifest as revisionedPath?